### PR TITLE
schemacrawler: Add version 16.16.5

### PR DESCRIPTION
**Pull request by sualeh**

### CODE REVIEW

**Comment by sualeh** [comment]
@tech189  - any ideas on why the validation is failing? The manifest works locally for me.

**Comment by sualeh** [comment]
@tech189  - are you fine with this change?

File: bucket/schemacrawler.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+  "version": "16.16.5",
+  "description": "SchemaCrawler - Free database schema discovery and comprehension tool",
```
Comment (tech189) [inline bucket/schemacrawler.json:3]:
```suggestion
  "description": "Database schema discovery and comprehension tool",
```
No need to repeat name in description.

**Review by tech189 — CHANGES_REQUESTED** [review]

File: bucket/schemacrawler.json (hunk 1 above)
Comment (sualeh) [reply bucket/schemacrawler.json]: I have removed the product name from the description but left the tag line is intact.

**Comment by tech189** [comment]
Hmm could you add the hash to autoupdate please? You publish it here: https://github.com/schemacrawler/SchemaCrawler-Installers/releases/download/v16.16.5/SchemaCrawler-16.16.5.msi.SHA256

**Comment by sualeh** [comment]
@tech189 - this one is a go too - what do you think?

File: bucket/schemacrawler.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  "license": "EPL-2.0",
+  "url": "https://github.com/schemacrawler/SchemaCrawler-Installers/releases/download/v16.16.5/SchemaCrawler-16.16.5.msi",
+  "hash": "89726fd5adbcb31cb0476873318e90f09b081c2ecd998a19ec6b0d197266aac4",
+  "extract_to": "schemacrawler-16.16.5",
+  "bin": "schemacrawler-16.16.5\\SchemaCrawler\\SchemaCrawler.exe",
+  "checkver": {
+    "github": "https://github.com/schemacrawler/SchemaCrawler"
+  },
+  "autoupdate": {
+    "url": "https://github.com/schemacrawler/SchemaCrawler-Installers/releases/download/v$version/SchemaCrawler-$version",
```
Comment (tech189) [inline bucket/schemacrawler.json:14]:
```suggestion
    "url": "https://github.com/schemacrawler/SchemaCrawler-Installers/releases/download/v$version/SchemaCrawler-$version.msi",
```
You forgot the extension, it's a 404 when manifest autoupdates

File: bucket/schemacrawler.json [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    "hash": {
+      "url": "https://github.com/schemacrawler/SchemaCrawler-Installers/releases/download/v$version/SchemaCrawler-$version.msi.SHA256",
```
Comment (tech189) [inline bucket/schemacrawler.json:16]:
```suggestion
      "url": "$url.SHA256",
```
Can be simplified!

**Review by tech189 — CHANGES_REQUESTED** [review]

**Comment by rashil2000** [comment]
- 64bit or 32bit?
- Why is extract_to required?

**Comment by sualeh** [comment]
> * 64bit or 32bit?
> * Why is extract_to required?

@rashil2000 
- I have added an architecture section, to be explicit about the bit version
- I have removed extract_to 
Please take a look. Thanks.

File: bucket/schemacrawler.json [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+  "description": "Free database schema discovery and comprehension tool",
+  "homepage": "https://www.schemacrawler.com/",
+  "license": "EPL-2.0",
+  "architecture": {
+    "64bit": {
+      "url": "https://github.com/schemacrawler/SchemaCrawler-Installers/releases/download/v16.16.5/SchemaCrawler-16.16.5.msi",
+      "hash": "89726fd5adbcb31cb0476873318e90f09b081c2ecd998a19ec6b0d197266aac4"
+    }
+  },
+  "bin": "SchemaCrawler\\SchemaCrawler.exe",
```
Comment (rashil2000) [inline bucket/schemacrawler.json:12]:
```suggestion
  "extract_dir": "SchemaCrawler",
  "bin": "SchemaCrawler.exe",
```

**Review by rashil2000 — CHANGES_REQUESTED** [review]
Reformat the file to use 4 spaces as tab instead of 2

File: bucket/schemacrawler.json (hunk 4 above)
Comment (sualeh) [reply bucket/schemacrawler.json]: That will not work, @rashil2000 - an extra SchemaCrawler directory is created by extracting the MSI.

Comment (sualeh) [reply bucket/schemacrawler.json]: Sorry - you are right. I am going to make that change. Do I need to make it in the updates section as well?

Comment (rashil2000) [reply bucket/schemacrawler.json]: Yes, this is for that directory itself. Try it.

**Comment by sualeh** [comment]
@rashil2000 - comments addressed in the latest checkin:

1. 4-space tabs
2. Use an extract dir

**Pull request merged**